### DISCUSSION TITLE
Aliases-related question

### DISCUSSION THREAD

**Discussion by wencywww** [docker-mailserver/docker-mailserver#2690] [Q&A]
Hi, coming from the vpopmail world i need ot ask:

1. Does DMS support domain aliases - for example, if i have a domain **example.com**, is it possible an alias domain to be created for it (**example1.com**) so that message to user@example1.com to be delivered to the existing user@example.com?
2. How to create an alias address for >1 real account - for example support@example.com to be an alias for both user1@example.com, user2@example.com ... userX@example.com?

Thanks in advance!

**Comment by wencywww** [comment]
Well, for the second question, i found the solution by chance:
`./setup.sh alias add support@example.com user1@example.com &&
./setup.sh alias add support@example.com user2@example.com`

Besides that, i found that this command permits creating an alias for a non-existing user email - so this command executes without errors althought user2@example.com does not exist:
`./setup.sh alias add support@example.com user2@example.com`

**Comment by georglauterbach** [comment]
Our official documentation is actually pretty straight on this :)

See [our documentation](https://docker-mailserver.github.io/docker-mailserver/edge/config/user-management/aliases/).

**Reply by wencywww** [reply]
Well, then i'm missing something there (provided that you mean my 1st question).
If i understood the docs well, for every real email address sitting under **example.com**, i must **_manualy_** add the appropriate alias for **example1.com** using `./setup.sh alias add ... `, right? 
Also, i have to say that the postfix docs mentinoed on the first line of your docs are confusing me ...

**Reply by georglauterbach** [reply]
> If i understood the docs well, for every real email address sitting under **example.com**, i must **_manualy_** add the appropriate alias for **example1.com** using `./setup.sh alias add ... `, right?

Yes :)

> Also, i have to say that the postfix docs mentinoed on the first line of your docs are confusing me ...

Raise an issue on the Postfix docs repo :)

**Reply by wencywww** [reply]
Well, thanks for your patience!
